source/faucet/main.d

```diff
@@ -348,6 +348,7 @@ public class Faucet : FaucetAPI
                     Amount result = a;
                     if (!a.add(b))
                         throw new Exception("Internal error while summing UTXOs");
+                    result.add(b);
                     return result;
```

Comment:
Ah, well, this whole hack is not needed anymore.
You can just replace that `fold` call with `sum`, since we added operator overload.

Reply:
Tested and works, will update.

Reply:
We can remove the `Exception` right?